server/assertions/selectors/selector_test.go

```diff
@@ -27,7 +37,8 @@ var pokeshopTrace = traces.Trace{
 			"tracetest.span.type": "http",
 			"http.status_code":    "201",
 		},
-		Name: "POST /import",
+		Name:      "POST /import",
+		StartTime: t(0),
```

Comment:
I'm not sure about this function to get the span time. I created it based on physics time variables: `t0`, `t1`, `t2`, etc.